pkg/api/pod/util_test.go

```diff
@@ -1517,12 +1396,6 @@ func TestDropNodeInclusionPolicyFields(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			if !test.enabled {
-				// TODO: this will be removed in 1.36
```

Comment:
todo is saying this should be removed in 1.36. Without going deeper, I would trust the TODO note and close this PR

/close